doc/xcodebuild.txt

```diff
@@ -1,31 +1,139 @@
-*xcodebuild.txt* Plugin to build and test Xcode projects from vim
+*xcodebuild.txt*
+
+                            xcodebuild.vim
+                Build and test Xcode projects from Vim
 
 ==============================================================================
-CONTENTS                               *xcodebuild-contents*
+CONTENTS                                                         *XCB-Contents*
 
+      1. About.......................... |XCB-About|
```

Comment:
I think every help section should start with `xcodebuild`, like [fugitive](https://github.com/tpope/vim-fugitive/blob/master/doc/fugitive.txt) does.

That way `:help xcodebuild<TAB>` shows everything, instead of only the parts that start with xcodebuild.

Maybe keep `XBuild` and `XTest` the same, though I think `:help XBuild<TAB>` will find it even if it's `xcodebuild-XBuild`.